src/monopoly/view/monopoly_view.js

```diff
@@ -883,71 +939,117 @@ MonopolyView.prototype.drawMessage = function() {
 
-    button1.interactive = true;
-    var context = this;
-    button1.click = function(mouseData){
-       this.turnMap.updateState("yes");
-    }.bind(this);
+    // button 2
+    this.button2 = new PIXI.Graphics();
+    this.button2Text = new PIXI.Text("",{});
+    this.addButton(this.button2, 250, 0xFF0000, container, "no_trade_clear", "No", this.button2Text);
 
-    this.button1Text = new PIXI.Text("Yes", {font: '30px Arial',
-                                                align : 'center',
-                                                wordWrap : true,
-                                                strokeThickness : .25,
-                                                //wordWrapWidth : (constants.tileLongSide - constants.tileColorLength),
-                                                wordWrapWidth : 150,
-                                                });
-    this.button1Text.x = 50;
-    button1.addChild(this.button1Text);
+    //button 3
+    this.button3 = new PIXI.Graphics();
+    this.button3Text = new PIXI.Text("",{});
+    this.addButton(this.button3, 500, 0x7A7A7A, container, "cancel", "Cancel", this.button3Text);
 
-    this.button2 = new PIXI.Graphics();
-    this.button2.x = 250;
-    this.button2.y = 180;
-    this.button2.beginFill(0xFF0000, 1);
-    this.button2.drawRect(0, 0, 200, 50);
-    container.addChild(this.button2);
-
-    this.button2.interactive = true;
-    this.button2.click = function(mouseData) {
-        this.turnMap.updateState("no");
-    }.bind(this);
-
-    this.button2Text = new PIXI.Text("No", {font: '30px Arial',
-                                                align : 'center',
-                                                wordWrap : true,
-                                                strokeThickness : .25,
-                                                //wordWrapWidth : (constants.tileLongSide - constants.tileColorLength),
-                                                wordWrapWidth : 150,
-                                                });
-    this.button2Text.x = 50;
-    this.button2.addChild(this.button2Text);
+    //button 4
+    this.button4 = new PIXI.Graphics();
+    this.button4Text = new PIXI.Text("",{});
+    this.addButton(this.button4, 750, 0xFFCC00, container, "add$", "+$20", this.button4Text, 20);
+
+    //button 5
+    this.button5 = new PIXI.Graphics();
+    this.button5Text = new PIXI.Text("",{});
+    this.addButton(this.button5, 1000, 0xFFCC00, container, "subtract$", "-$20", this.button5Text, -20);
 
-    this.stage.addChild(container);
 };
 
+MonopolyView.prototype.addButton = function(btn, x, color, container, ans, text, btnText, money){
+    btn.x = x;
+    btn.y = 180;
+    btn.beginFill(color, 1);
+    btn.drawRect(0, 0, 200, 50);
+    container.addChild(btn);
+    var context = this;
+    btn.interactive = true;
+    btn.click = function(mouseData) {
+        if(money){
+            context.game.trade.money += money;
```

Comment:
we should be calling methods of the game to update this and not doing the update itself in the view